lib/main.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+const log = require('./log');
+const loader = require('taskcluster-lib-loader');
+const _ = require('lodash');
+const {initCloudWatchEventListener} = require('./cloud-watch-event-listener');
+const aws = require('aws-sdk');
+const monitor = require('taskcluster-lib-monitor');
+const config =require('typed-env-config');
+const setup = require('./cloud-watch-ingest');
+const PG = require('pg');
+const Pool = require('pg-pool');
+const {State} = require('./state');
+const url = require('url');
+const assert = require('assert');
+
+const types = PG.types;
+
+// Since Postgres returns 64-bit ints from certain operations,
+// we want to parse them down to a JS number.  Yes, we're loosing
+// precision here, but the chances of us having any integer that's
+// greater than ~53-bits here tiny
+types.setTypeParser(20, x => parseInt(x))
+
+//PG.defaults.ssl = true;
```

Comment:
so that's actually commented because it didn't work after I switched to using the `pg-pool` module.  I will make sure that SSL is enabled where appropriate, but since this is running on an internal heroku network, SSL is not *superduper critical*, but obviously preferable.